CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Most recent change on the bottom.
 
 ## [Unreleased]
 
+## [0.3.3] - 2021-06-24
+### Added
+- `to_ase` method in `AtomicData.py` to convert `AtomicData` object to (list of) `ase.Atoms` object(s)
+- Two unit tests in `test_AtomicData.py` to test the above
```

Comment:
```suggestion
```

In general, the goal of a changelog is to have a human-readable record of things that are relevant to users (bugs found, fixed; features added) — so usually we don't list tests or other things that are done only for development purposes